### PR TITLE
AEA-2743 Log additional request payload identifiers

### PR DESCRIPTION
**Pull request by StefanoFrazzetto**
**Changes**:
- Log identifiers for incoming payloads:
  * Payload identifier
  * Patient NHS number
  * Sender ODS Code
  * Prescription short form ID
- Added new package for customising Jest tests expects message
- Added missing logging for incoming payloads hash
- Created tests to ensure presence of log messages and validity of their contents
- Fixed NodeJS runner version in VS Code launch settings

### CODE REVIEW

File: coordinator/src/routes/logging.ts [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+    super(new FhirPathBuilder().claim())
+  }
+
+  getPayloadIdentifier = (): string => this.builder.identifier()
+  getNhsNumber = (): string => this.builder.patient().nhsNumber()
+  getPrescriptionNumber= (): string => this.builder.prescription().shortFormId()
+  getOdsCode = (): string => this.builder.organization().odsCode()
+}
+
+// TODO: Add examples for single patient and bulk release
```
Comment (AndyMather24) [inline coordinator/src/routes/logging.ts:75]: Hi @StefanoFrazzetto, have we got a ticket to cover this "TODO"?

File: coordinator/tests/resources/test-resources.ts [hunk 2]
```diff
@@ -111,6 +114,27 @@ export class ExamplePrescription {
       const hl7V3MessageClaimStr = fs.readFileSync(hl7V3MessageClaimPath, "utf-8")
       this.hl7V3MessageClaim = XmlJs.xml2js(hl7V3MessageClaimStr, {compact: true})
     }
+
+    // TODO: Add more examples
```
Comment (AndyMather24) [inline coordinator/tests/resources/test-resources.ts:118]: TODO's ?

File: coordinator/src/routes/logging.ts (hunk 1 above)
Comment (StefanoFrazzetto) [reply coordinator/src/routes/logging.ts:75]: As discussed earlier, we might have more examples and/or have plans to add more end-to-end examples. Leaving the TODO for posterity, as a reminder for this.

**Review by AndyMather24 — APPROVED** [review]
Great work!

**Pull request merged**